resources/deprecations.properties

```
@@ -125,7 +135,12 @@ origo-issue-notifier = https://wiki.jenkins.io/x/qgabAg
 paranoia = https://git.io/JfaQS
 # https://plugins.jenkins.io/perfectomobile/
 perfectomobile = https://developers.perfectomobile.com/display/PD/Legacy+%7C+Jenkins+plugin
+perforce = https://jenkins.io/security/advisory/2018-03-26/
+persona = https://www.jenkins.io/security/plugins/#suspensions
 php = https://issues.jenkins.io/browse/INFRA-2487
+# https://github.com/jenkins-infra/update-center2/pull/384
+play-autotest-plugin = https://git.io/JsmWA
+pipeline-classpath = https://jenkins.io/security/advisory/2017-03-20
```

Updated the suspensions list, so now:

```suggestion
pipeline-classpath = https://www.jenkins.io/security/plugins/#suspensions
```